packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,264 @@
+# Terra Application Layout
+
+## Usage Summary
+
+1. The ApplicationLayout requires a `Base` component (from `terra-base`) and any `Router` component (from `react-router-dom`) in its parent hierarchy.
```

Comment:
Could we list these props as a table with their prop types, if required and defaults?

Reply:
That's basically already there on the About page, it's just full of custom prop types that don't help people. I made another table with the actual prop information spelled out but the formatting got atrocious. I can make a table with very little information and leave the sections below to fill in the blanks; do you think that'd be helpful?